contracts/cw1155-base/src/contract.rs

```diff
@@ -233,9 +233,9 @@ pub fn execute_mint(
     }
 
     // insert if not exist
-    let key = TOKENS.key(&token_id);
-    if deps.storage.get(&key).is_none() {
-        key.save(deps.storage, &"".to_owned())?;
+    if !TOKENS.has(deps.storage, &token_id) {
```

Comment:
👍🏼

Reply:
When writing tests I found out the (untested) case would actually panic.
If you attempt to write `b""` to the db.

(Which we added to avoid getting confusing results from the cosmos sdk storage layer). So I fixed that with real data as well.

Reply:
So, the original code had that flaw too?

Reply:
Oh, I guess you mean, as a key, not as a value.